### PR TITLE
Add section to documentation about time tracking.

### PR DESCRIPTION
**Pull request by lydiagu**

### CODE REVIEW

File: docs/source/features.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
+###################
+Feature Walkthrough
+###################
+
+Below, we'll walk through the various features in Orchestra.
+
+**********************
```
Comment (marcua) [inline docs/source/features.rst]: delete for now

File: docs/source/features.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+timecard page where workers can update and manually create time entries.
+
+Timer
+=====
+
+The navigation bar in the task dashboard and task pages has a timer icon.
+
+.. image:: ../static/img/time_tracking/timer_icon.png
+
+When clicked on, a dropdown appears that allows the worker to toggle a timer
```
Comment (marcua) [inline docs/source/features.rst]: "When clicked on," -> "If you click on the timer icon,"

File: docs/source/features.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+to track their work time.
+
+.. image:: ../static/img/time_tracking/timer_dropdown.png
+
+The worker can optionally add a description and specify the task they are
+working on.
+
+When the timer is stopped, a time entry is automatically created for the
+amount of time tracked. A worker can go to the timecard page to edit and
```
Comment (marcua) [inline docs/source/features.rst]: "time tracked" -> "time worked"

File: docs/source/features.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+manually add time entries.
+
+Timecard
+========
+
+The timecard page contains a list of time entries for the past week, grouped
```
Comment (marcua) [inline docs/source/features.rst]: "past week" won't be true once there's filtering

File: docs/source/features.rst [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+by date.
+
+.. image:: ../static/img/time_tracking/timecard.png
+
+The time to the right of the date shows the total time worked that day.
+
+A time entry can be manually added (versus automatically added by stopping the
+timer), by clicking the plus icon next to each date.
```
Comment (marcua) [inline docs/source/features.rst]: no comma

File: docs/source/features.rst [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+======
+
+- Time entries automatically created when the timer is stopped have the date
+  set to the current UTC timestamp date. This means that the date might be
+  different than the date in the worker's time zone. In general, handling time
+  entries across multiple time zones is difficult, and we are still working on
+  a better user experience.
+- Time entries automatically created when the timer is stopped have the work
+  time set to the hours:minutes:seconds displayed in the timer. However, the
+  timecard page only shows hours:minutes for time entries for readability.
```
Comment (marcua) [inline docs/source/features.rst:71]: We will be rounding up---we should mention that.

Comment (lydiagu) [inline docs/source/features.rst:71]: isn't that what i say on the next line?

**Pull request merged**